Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3-stretch
+MAINTAINER Lars von Wedel <vonwedel@me.com>
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY requirements.txt requirements.txt
+RUN pip install --upgrade pip
+RUN pip install --no-cache-dir -r requirements.txt
+
+RUN apt-get update && apt-get install -y bluez
+
+COPY . .
+
+CMD [ "python3", "./miflora-mqtt-daemon.py", "--config_dir", "/config" ]
```

Comment:
Where does `/config come from`? Maybe I missed something related to docker

Reply:
When running the container the /config directory can be bound to a "real" directory on a docker host. This is actually one of the nice things about docker - I have all config files for docker apps in /srv/docker/... and bind the respective directories to whatever the application uses inside the container. I run miflora bridge with command `docker run -it --name miflora-mqtt-daemon -v /srv/docker/miflora-mqtt-daemon/:/config miflora-mqtt-daemon`. I can add some hints to the README on this.

Reply:
I know how docker works but there was no hint how your `/config` is bound